src/lib/serviceRoutinesV2/getEntity.h

```diff
@@ -0,0 +1,49 @@
+#ifndef SRC_LIB_SERVICEROUTINES_GETENTITYV2_H_
+#define SRC_LIB_SERVICEROUTINES_GETENTITYV2_H_
```

Comment:
The name to use for the define is:

    SRC_LIB_SERVICEROUTINESV2_GETENTITY_H_

(in three places)